src/main/java/seedu/fridgefriend/utilities/Save.java

```diff
@@ -0,0 +1,84 @@
+package seedu.fridgefriend.utilities;
+
+import seedu.fridgefriend.exception.InvalidDateException;
+import seedu.fridgefriend.food.Food;
+import seedu.fridgefriend.food.FoodCategory;
+import seedu.fridgefriend.food.FoodStorageLocation;
+import seedu.fridgefriend.food.Fridge;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+public class Save {
```

Comment:
Class name should be a noun not a verb, can we name this Storage instead?

Reply:
That conflicts with Food Storage Location, and save can be a noun anyway